lib/domain/libBlockchain/langs/rust/near/NearModel.h

```
@@ -61,6 +61,13 @@ namespace vanguard {
             return analysis::InterProceduralAnalyses<Domain>::readsMemFrom(&ins, selfVal);
         }
 
+        std::unordered_set<Variable<Domain> *> getStorageReads(typename Domain::Instruction &ins) const override {
+            return {};
```

Should we maybe throw a runtime exception?